src/main/ui/app/component/NamedList.jsx

```diff
@@ -27,9 +27,8 @@ export default class NamedList extends Component {
         const COLUMNS_IF_SIZE = 5;
 
         const list = this.props.list;
-        const colSize = list.addOns.length > COLUMNS_IF_SIZE ?
```

Comment:
@djazayeri I'd say we could wait for a reply. If we don't get one then we'll go with removing the feature . Honestly, looking at the screenshots the 2-column version looks good as well but the only thing is as to whether the output is worth the effort. If we go with the 2-sided version here then there are more places such as list of downloads , search results etc which could use this as well.